.github/workflows/ci.yml

```diff
@@ -56,16 +56,16 @@ jobs:
               run: python setup.py build
               shell: bash
               env:
-                CC: gcc-5
-                CXX: g++-5
+                CC: gcc-7
+                CXX: g++-7
               if: ${{ matrix.compiler == 'gcc' }}
 
             - name: Build package
               run: python setup.py build
               shell: bash
               env:
-                CC: clang-6.0
-                CXX: clang++-6.0
+                CC: clang-9
+                CXX: clang++-9
```

Comment:
clang 8 should still be available? It would be good to test whatever minimum version we can get our hands on.

Reply:
According to https://github.com/actions/virtual-environments/blob/main/images/linux/toolsets/toolset-1804.json#L243, only version 9 is installed.

And Ubuntu 18.04 has clang 6, but I don't know how to use this one: https://packages.ubuntu.com/bionic/clang

Reply:
Ah, indeed. For the osm2pgsql tests we [install the packaged clang](https://github.com/openstreetmap/osm2pgsql/blob/master/.github/actions/ubuntu-prerequisites/action.yml).

Reply:
I've gone back to clang-6.0 (the default for Ubuntu 18.04), by adding:
```
sudo apt-get install -y -qq clang-6.0
```